.github/workflows/validate.yaml

```diff
@@ -0,0 +1,62 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: Auto Validate Quickstarts
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
What about tags?

Reply:
Sure, we can run validation on a tag build.